packages/react-portal/src/components/Portal/Portal.types.ts

```diff
@@ -10,6 +10,13 @@ export interface PortalProps {
    * @defaultValue a new element on document.body without any styling
    */
   mountNode?: HTMLDivElement | null;
+  /**
+   * Has an affect only when server side rendering is used. Indicates that a Portal will be rendered only on client and
+   * it's fine to skip initial null render. Useful for Dialogs and similar components that are never rendered initially.
+   *
+   * @defaultValue false
+   */
+  isRenderedOnlyOnClient?: boolean;
```

Comment:
It might be clearer to use invert the prop and set a `true` default value

```suggestion
  /**
   * This component supports Server Side Rendering (SSR) with a null render on the server
   * For components like Dialogs which are never rendered initially set this to `false` to skip the initial null render
   * NOTE: This prop has no effect on non-SSR apps
   *
   * @defaultValue false
   */
  requireServerRender?: boolean;
```

Reply:
TBH I don't know what is a better option there 😅 @miroslavstastny @behowell do you have any suggestions?

Reply:
I removed this for now, we can add it later if it will become an issue for perf.